tuf/client/updater.py

```diff
@@ -1561,85 +1659,73 @@ def _update_metadata_if_changed(self, metadata_role,
 
 
 
-  def _fileinfo_has_changed(self, metadata_filename, new_fileinfo):
+  def _versioninfo_has_changed(self, metadata_filename, new_versioninfo):
```

Comment:
See comments below. The method isn't really checking for *change*, but *increase* in version number. It should perhaps be called `_file_has_been_updated`.